lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.ts

```diff
@@ -115,39 +116,65 @@ export class StartProcessCloudComponent implements OnChanges, OnInit, OnDestroy
     processDefinitionSelection: EventEmitter<ProcessDefinitionCloud> = new EventEmitter<ProcessDefinitionCloud>();
 
     processDefinitionList: ProcessDefinitionCloud[] = [];
-    processDefinitionCurrent: ProcessDefinitionCloud;
+    processDefinitionCurrent?: ProcessDefinitionCloud;
     errorMessageId: string = '';
-    processForm: UntypedFormGroup;
     processPayloadCloud = new ProcessPayloadCloud();
     filteredProcesses: ProcessDefinitionCloud[] = [];
-    isLoading = false;
-    isFormCloudLoaded = false;
-    formCloud: FormModel;
     staticMappings: TaskVariableCloud[] = [];
-    resolvedValues: TaskVariableCloud[];
+    resolvedValues?: TaskVariableCloud[];
 
     protected onDestroy$ = new Subject<boolean>();
+
+    isProcessStarting = false;
+    isFormCloudLoaded = false;
+    isFormCloudLoading = false;
     processDefinitionLoaded = false;
-    loading$ = new BehaviorSubject<boolean>(!this.processDefinitionLoaded);
 
-    constructor(
-        private startProcessCloudService: StartProcessCloudService,
-        private formBuilder: UntypedFormBuilder,
-        private processNameCloudPipe: ProcessNameCloudPipe
-    ) {}
+    formCloud?: FormModel;
+    processForm = new FormGroup({
+        processInstanceName: new FormControl('', [
+            Validators.required,
+            Validators.maxLength(this.getMaxNameLength()),
+            Validators.pattern('^[^\\s]+(\\s+[^\\s]+)*$')
+        ]),
+        processDefinition: new FormControl('', [Validators.required, this.processDefinitionNameValidator()])
+    });
```

Comment:
Is it the better approach to no use FormBuilder service?

Reply:
It depends. FormBuilder gives you more possibilities (e.g. nonNullableForm, simplified grouping) but for basic usage you can simply use new FormGroup.
https://angular.dev/guide/forms/reactive-forms#create-a-formgroup-instance 
https://angular.dev/guide/forms/reactive-forms#using-the-formbuilder-service-to-generate-controls

Reply:
Makes sense, we will save one inject() with this approach so we'll benefit on that, resolved